src/Libraries/CoreNodes/DSCoreNodesImages.resx

```diff
@@ -5119,44 +5119,6 @@
         gOTfKGA1R7m+SSE8APXA8bsfLqezAAAAAElFTkSuQmCC
 </value>
   </data>
-  <data name="DSCore.IO.File.ReadText.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
Looks like this change is not relevant, if there was no change, can you please remove this file from the PR.